src/routes/estimations/estimations.controller.ts

```diff
@@ -0,0 +1,29 @@
+import { Body, Controller, HttpCode, Param, Post } from '@nestjs/common';
+import { ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import { EstimationRequest } from './entities/estimation-request.entity';
+import { EstimationResponse } from './entities/estimation-response.entity';
+import { EstimationsService } from './estimations.service';
+
+@ApiTags('estimations')
+@Controller({
+  path: '',
+  version: '2',
+})
+export class EstimationsController {
+  constructor(private readonly estimationsService: EstimationsService) {}
+
+  @ApiOkResponse({ type: EstimationResponse })
```

Comment:
Isn't' this inferred from the route signature? 🤔

Reply:
> I tried to remove it, but it doesn't get inferred. I'm going to investigate it further in the Nest docs, but it seems required (we have this kind of annotations in other controllers as well) 🤔

Nevermind then! I might have mixed up some concepts. I will need to double check the docs!